src/ragger/conftest/configuration.py

```diff
@@ -0,0 +1,23 @@
+from typing import Dict, Any
+
+# The following parameters, if any, are required.
+# Please set their values in your 'conftest.py' file
+REQUIRED: Dict[str, Any] = {}
```

Comment:
As there is nothing `REQUIRED` for now, I'm in favor of removing all the related code. It's just noise.

Reply:
Ok